addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/__tests__/environment-sc-service.test.js

```diff
@@ -605,6 +605,63 @@ describe('EnvironmentSCService', () => {
       expect(storageGatewayService.updateStudyFileMountIPAllowList).not.toHaveBeenCalled();
     });
 
+    it('should truncate excessively long error messages and succeed to update', async () => {
```

Comment:
Would this test fail if you didn't have the truncate fix? I am not sure where we are testing if the truncate happened in this test before the ddb update.

Reply:
yes, it will fail at: https://github.com/awslabs/service-workbench-on-aws/blob/70d989172013fe9213194723e791f230d730ca6f/addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js#L458

That because the validation schema has a mak length of 2048:
https://github.com/awslabs/service-workbench-on-aws/blob/b799b5bfb1305c07b04742725697c6b0e5f47bdc/addons/addon-base-raas/packages/base-raas-services/lib/schema/update-environment-sc.json#L25-L27